### DISCUSSION TITLE
Pregnancy model no renal clearance

### DISCUSSION THREAD

**Discussion by YlvaJohansson** [Open-Systems-Pharmacology/Forum#1791] [Questions & Problems]
Hi,
I have an issue regarding creating a pregnancy model using Pk-Sim and MoBi. I have followed the Dallman et al., 2017 tutorial and understood that when I take my model from PK-Sim to MoBi, my renal clearance has to be added again. I do this for my parent compound as well as the metabolite and I can see that the process is followed back to PK-Sim when I run my population simulation:
![Capture](https://github.com/Open-Systems-Pharmacology/Forum/assets/56597996/5f36c3b5-fe4e-41b6-a864-43119fb94448)

I can also see in PK-Sim that the parameters fit and that the reaction diagram is correct. However, I have no excretion of my metabolite when I simulate the model. 

Anyone who has had the same issues?

best,
Ylva

**Comment by MSCUKW** [comment]
Hey, 
I also try to establish a pregnancy model with a drug and its metabolite; however, I have problems to include the renal clearance of my metabolite. I do not find how to define the specific clearance for the metabolite, as there is only one "specific renal clearance" in the Parameter window, even if I chose both substances in the Molecule window (by the "Add Molecule" option)..... 
How did you include the specific clearance of your metabolite? Did that work? Maybe that´s the problem with your simulation?
Best

**Reply by YlvaJohansson** [reply]
Hej,

So what I did is that I added my renal clearance for my parent compound and specified the parameter. Then I saved that process as a pklm file and uploaded it and changed it to my metabolite. So I have two renal clearance processess. However, none of them are simulated in urine later in PK-Sim.... 

I am trying to see how they have done it before on the MoBi and PK-Sim files uploaded here!
I'll update if i manage to figure it out. 


Best,
Ylva

**Reply by YlvaJohansson** [reply]
Now it works for me, probably  mistake I did in PK-Sim before sending to MoBi. So if you follow my sugestion there on how to add two processess it should work :)

**Reply by MSCUKW** [reply]
Thanks a lot for the information and your help. 
However, I´m again in trouble, even if I´m following your protocol. When I upload the .pklm file and change it to my metabolite, I do not create a second renal clearance process, but overwrite the first one (with my parent compound). I cannot figure out, where is the trick, with inlcuding a second renal clearance process.....

**Reply by MSCUKW** [reply]
Now, I figured it out!! Thanks a lot for your help!!!!
